handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,57 @@
+# MinIO as storage provider for the FileService
+
+SystemLink FileService allows to configure and usethe Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the FileService also works with MinIO which is a Amazon S3 compatible server-side software storage stack.
+
+The minimum required SystemLink version is 2020 R3.
+
+In order to use MinIO, there are two steps required. First, setting up the MinIO server and, second, configuring the FileService to use that server.
+
+## Running MinIO server
```

Comment:
Consider including numbered steps in this section and the next. That way, it's easier for people to follow.

I'd also check with @GrKuehnH to avoid content duplication. I think we're fine for this one, but just as a best practice : )

Reply:
I think numbered steps for the first section is fine. I'm hesitant to do so for the second section, since this is just one step after all. If numbered steps is preferred, I'd rather re-write the section. Would you like me to provide an alternative phrasing for this section that is numbered? @jjaco2

Reply:
This looks good! Adding a few tweaks here and there, just for NI style and stuff. For the second section, I agree with you looking at it again. Below I've suggested a change that could make the second section more direct while also not having to number anything -- so minimal rewriting on your part!